wasm/src/main/java/com/dylibso/chicory/wasm/Parser.java

```diff
@@ -167,17 +168,18 @@ void parse(InputStream in, ParserListener listener) {
                     "unexpected token: unsupported version, found: " + version + " expected: " + 1);
         }
 
+        var firstTime = true;
```

Comment:
I think we should leave a comment for our future selves on this "trick", but I'm ok with it if there is no better way ...